app/main/RTMedia.php

```diff
@@ -1496,6 +1496,49 @@ function rtmedia_get_site_option( $option_name, $default = false ) {
 	return $return_val;
 }
 
+function rtm_privacy_message_on_website() {
+	global $rtmedia;
+	$options = $rtmedia->options;
+
+	if( "1" === $options['general_upload_terms_show_pricacy_message'] && empty( $_COOKIE[ 'rtm_show_privacy_message' ] ) ) {
+		echo "<div class='privacy_message_wrapper'><p>" . wp_kses_post( $options[ 'general_upload_terms_privacy_message' ] ) . "</p><span class='dashicons dashicons-no' id='close_rtm_privacy_message'></span></div>";
```

Comment:
Is `$options[ 'general_upload_terms_privacy_message' ]` will be containing HTML tags?

Reply:
Yes.

Reply:
Okay.